README.md

```diff
@@ -57,14 +56,8 @@ Alternatively, you can manually set up your dev environment outside of make by p
 cd plugin
 yarn install
 
-# If necessary, make sure you change the "KIALI_PROXY" value in .env.development so it points to your Kiali Server URL
+# If necessary, make sure you change the "API_PROXY" value in .env.development so it points to your Kiali Server URL
 # vi .env.development
-
-# Copy the plugin-config.json file into the "dist" folder to emulate the ConfigMap in a local environment
-cp plugin-config.json dist
```

Comment:
I would leave this here - maybe add to the comment saying, `This config file is empty today so is not really needed. This is here for future enhancement.`